.eslintrc.json

```diff
@@ -79,22 +86,16 @@
         "@typescript-eslint/no-namespace": "off",
         "@typescript-eslint/no-non-null-asserted-optional-chain": "off",
         "@typescript-eslint/no-var-requires": "off",
+        "@typescript-eslint/no-empty-interface": "off",
```

Comment:
This one deserves an explanation; while it's definitely bad to have empty interfaces, we seem to have quite a few cases which are "valid" in that they do things like rebinding another interface with new type variables (many in `types`), providing an interface that is augmented elsewhere (`SourceFile`), and so on.